abb_irb120_support/urdf/irb120_macro.xacro

```diff
@@ -0,0 +1,195 @@
+		</collision>
+	</link>
+	<link name="${prefix}link_1">
+		<visual>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/visual/link_1.STL"/>
+			</geometry>
+            <material name="abb_orange"/>			
+		</visual>
+		<collision>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/collision/link_1.STL"/>
+			</geometry>
+            <material name="yellow"/>
+		</collision>
+	</link>
+	<link name="${prefix}link_2">
+		<visual>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/visual/link_2.STL"/>
+			</geometry>
+            <material name="abb_orange"/>
+		</visual>
+		<collision>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/collision/link_2.STL"/>
+			</geometry>
+            <material name="yellow"/>
+		</collision>
+	</link>
+	<link name="${prefix}link_3">
+		<visual>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/visual/link_3.STL"/>
+			</geometry>
+            <material name="abb_orange"/>
+		</visual>
+		<collision>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/collision/link_3.STL"/>
+			</geometry>
+            <material name="yellow"/>
+		</collision>
+	</link>
+	<link name="${prefix}link_4">
+		<visual>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/visual/link_4.STL"/>
+			</geometry>
+            <material name="abb_orange"/>
+		</visual>
+		<collision>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/collision/link_4.STL"/>
+			</geometry>
+            <material name="yellow"/>
+		</collision>
+	</link>
+	<link name="${prefix}link_5">
+		<visual>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/visual/link_5.STL"/>
+			</geometry>
+            <material name="abb_orange"/>
+		</visual>
+		<collision>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/collision/link_5.STL"/>
+			</geometry>
+            <material name="yellow"/>
+		</collision>
+	</link>
+	<link name="${prefix}link_6">
+		<visual>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/visual/link_6.STL"/>
+			</geometry>
+			<material name="black">
+			    <color rgba="0 0 0 1"/>
+			</material>	
+		</visual>
+		<collision>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/collision/link_6.STL"/>
+			</geometry>
+            <material name="yellow"/>
```

Comment:
this seems like it's not correctly indented (also for other material elements earlier)?